### DISCUSSION TITLE
Option to disable Discord button

### DISCUSSION THREAD

**Discussion by Tea23** [LizardByte/community#520] [Sunshine] [answered]
### Select Topic Area

Feature Request

### Body

I am evaluating Sunshine for classroom use. We might not go with it (RDP might be *just* enough), but if we do, we'll be deploying sunshine on a bunch of VMs and some student-level setup is going to be needed, where they open the Sunshine webui to enter the PIN from Moonlight.

I noticed that the Discord button in the bottom right can be opened to instantly show the Discord server, and every so often a message from the server is displayed as a notification, even if it's not open. 

I'm sure nothing particularly inappropriate will pop up (and this is in a University environment anyway), but the notifications could be distracting and students might be tempted to just explore the Discord instead of setting up for their assignment.

The option to disable the integration via config file could be worthwhile in this instance.

**Comment by ReenigneArcher** [comment]
Just FYI, Sunshine is a game streaming application, not remote desktop tool. You really should go with RDP if you want to control the security of the environment with proper ACLs.

The notification is a random video game quote after 7 minutes, which comes from https://github.com/LizardByte/uno/blob/master/gh-pages-template%2Frandom-quotes%2Fgames.json ... No other messages cause a notification in the widget.

The safe (without profanity) quote is used.

You're free to patch out the widgetbot crate or block the requests it makes, but I think it's unlikely to be configurable anytime soon.

**Reply by Tea23** [reply]
Absolutely it is, this is an unorthodox application. It's for a chemistry course where 3D acceleration is needed to render molecules (or something - I just do what I am told!). Early RDP tests gave us enough input lag to be disruptive, so exploring this was the result of a eureka moment.

I don't think there is any non-game oriented implementation of what I'm being asked for here that can take full advantage of NVENC. Sunshine truly does suffice.

Thanks for clarifying the notification source - since it was presented as a message notification it made sense that it might have been coming *from* the server.

**Reply by ReenigneArcher** [reply]
> where 3D acceleration is needed

That makes more sense why you're exploring Sunshine/Moonlight then. You probably won't find better performance than using this combination.

P.S. I'm not rejecting this feature request or anything, just don't when I can get to it.

**Comment by ReenigneArcher** [comment, marked as answer]
Widgetbot is being removed in https://github.com/LizardByte/Sunshine/pull/4630
